README.md

```diff
@@ -56,3 +58,14 @@ to reflect these differences.
 The code and our documentation are the ultimate sources of truth. CFPs are merely
 the starting point for the implementation.
 
+# Status
+
+ The status of a CFP indicates where it is in the process. The CFP process is as follows:
+
+ 1. ``Provisional``: All new PRs are given this status. This CFP's design has not yet been approved.
+ 2. ``Implementable``: Approved PRs that are merged are given this status. Their CFP design has been approved, and now the work can be started to implement them.
+ 3. ``Experimental``: The CFP has been implemented and is now in a 'Beta' state.
+ 4. ``Stable``: The CFP has been implemented, rigororously tested, and is now in a 'GA' state.
```

Comment:
I guess related to the previous comment, we have the state of the CFP then the state of the feature. Each of these lines seems to describe the state of the CFP as tracking a specific state of the feature, though these terms like 'beta' or 'GA' are not currently defined anywhere, so it's a little hard to understand what those mean.

Is the rigorous testing intended for the _design_ or the _implementation_?